index.html

```diff
@@ -9,47 +9,73 @@
     </head>
     <body>
         <header>
-            <h1>What Cocktail Should I Make?</h1>
-            <img
-                src="https://images.unsplash.com/photo-1611873027887-19a588803816?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=627&q=80"
-                alt="#"
-            />
-            <form>
-                <label for="alcohol">Got an alcohol of choice?</label>
-
-                <select name="alcohol" id="alcohol-select">
-                    <option value="">Choose your booze</option>
-                    <option value="bourbon">Bourbon</option>
-                    <option value="brandy">Brandy</option>
-                    <option value="cognac">Cognac</option>
-                    <option value="gin">Gin</option>
-                    <option value="rum">Rum</option>
-                    <option value="scotch">Scotch</option>
-                    <option value="tequila">Tequila</option>
-                    <option value="triplesec">Triple Sec</option>
-                    <option value="vodka">Vodka</option>
-                    <option value="whiskey">Whiskey</option>
-                </select>
-            </form>
+            <div class="wrapper">
+                <!-- Can change the colours of the cocktail image depending on our colour palette for teh app: -->
+                <img src="./styles/assets/cocktailImage.svg"
+                    alt="A vector image of a cute sparkly orange cocktail in a soft blue glass" />
```

Comment:
I love your image!

Reply:
Sweet! We can mess around with the colours later but thought it would be fun!